src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnRunner.java

```diff
@@ -193,7 +205,11 @@ public SpawnResult exec(Spawn spawn, SpawnExecutionPolicy policy)
       }
 
       try {
-        return downloadRemoteResults(result, policy.getFileOutErr());
+        SpawnResult r = downloadRemoteResults(result, policy.getFileOutErr());
+        if(stats != null) {
+          stats.addRemoteExec();
```

Comment:
I think it's confusing that you are counting only successful executions, for some definition of successful that will not be clear to the user. I think it will be simpler if you count all remote and local executions explicitly, even if you need to add a counter for fallback (you can just increment it into execLocallyOrFail, probably).

My other concern is that this might not be what Nick had in mind. This will only be enabled at all if remote execution or one form of remote caching is enabled (but not the other, --experimental_remote_spawn_cache). I suspect that Nick wants (a) cache hit ratio for the --experimental_remote_spawn_cache option as well (b) local cache hit ratio (c) stats for local execution only with no remoting. Better check with him just in case.